packages/code-studio/src/styleguide/StyleGuide.tsx

```diff
@@ -31,19 +31,22 @@ import { GoldenLayout } from './GoldenLayout';
 import { RandomAreaPlotAnimation } from './RandomAreaPlotAnimation';
 import SpectrumComparison from './SpectrumComparison';
 
-const stickyProps = {
-  position: 'sticky',
-  justifyContent: 'end',
-  zIndex: 1,
-  UNSAFE_style: {
-    float: 'right',
-  },
-} as const;
-
 function StyleGuide(): React.ReactElement {
-  const isolateSection = window.location.search.includes('isolateSection=true');
+  const isTestMode = window.location.search.includes('testMode=true');
```

Comment:
I would make this case insensitive with something like `search.toLowerCase().includes('testmode=true')`